src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,377 @@
+community-based approach attempts to focus on what's best for its developers, focusing on the things that they need
+to write great mods while also taking the bigger picture into account, and avoiding playing favourites with larger 
+mods or more well-known mod developers.
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}What about the Fabric mods I already play?{% endsummary %}
+
+If you already use Fabric, we have good news -- Quilt is also compatible with most Fabric mods! As long as the mods
+don't make use of internal Fabric APIs, you should be able to add them to your `mods` folder as you always have and,
+assuming you have our Quilted Fabric API installed (which is part of QSL), things should just work.
+
+We plan on maintaining Fabric support for as long as we can -- we'll only stop it if it becomes infeasible to maintain
+it as part of Quilt itself. That said, if it does happen, we'll be gifting that part of the project to the community,
+and providing its new maintainers with as much help as we can.
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}Won't this just further split the community?{% endsummary %}
+
+This is a question we get a lot, and one that's clearly on a lot of people's minds. Instead of trying to predict the
+future, though, let's break this one down a bit.
+
+The concept of Minecraft modding has existed almost as long as Minecraft itself has -- with a very diverse range of
+mods, mod-loaders and communities being created and finding their niches. That said, one mod-loader ended up taking
+the crown for a very long time -- Forge.
+
+As Forge's popularity grew and its range of mods expanded, many users became used to their workflow: they'd install
+Forge, find some mods and, since Forge was the only really viable mod-loader for a long time, those mods probably
+ just worked. Later on, other mod-loaders started appearing and disappearing. Liteloader showed 
+up with its client-side modding framework. Rift appeared, and then died. Fabric appeared, and started gaining 
+momentum -- and that's when people _really_ started getting upset.
+
+Since then, there's been a lot of complaints about additional mod-loaders "fracturing the community". We don't have
+a crystal ball, but of course the concept has some merit: as more mod-loaders pop up and more mods appear, it's
+inevitable that users are going to run into mods that they want to play with, but that are incompatible with their 
+favourite mod-loader. This can certainly be annoying, but it doesn't merit the harassment 
+that many mod developers have gotten for refusing to bring their mods to a different mod-loader. We also believe in
+the benefits that come with providing users with more choice -- for example, motivating every project to do better, or 
+the extra room for technical and modding innovation.
```

Comment:
this wording more clearly connects with the thought of harassment, making it seem less like a random accusation
```suggestion
that many mod developers have gotten for refusing to bring their mods to a different mod-loader. We believe that
the benefits that come with providing users with more choice -- for example, motivating every project to do better, or 
the extra room for technical and modding innovation -- outweigh the inconveniences of fragmentation.
```